homeassistant/components/fortios/device_tracker.py

```diff
@@ -0,0 +1,99 @@
+"""
+Support to use FortiOS device like FortiGate as device tracker.
+
+This component is part of the device_tracker platform.
+"""
+import logging
+import voluptuous as vol
+
+from fortiosapi import FortiOSAPI
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.device_tracker import (
+    DOMAIN, PLATFORM_SCHEMA, DeviceScanner)
+from homeassistant.const import CONF_HOST, CONF_TOKEN
+from homeassistant.const import CONF_VERIFY_SSL
+
+_LOGGER = logging.getLogger(__name__)
+DEFAULT_VERIFY_SSL = False
+
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Required(CONF_TOKEN): cv.string,
+    vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean
+})
+
+
+def get_scanner(hass, config):
+    """Validate the configuration and return a FortiOSDeviceScanner."""
+
+    host = config[DOMAIN][CONF_HOST]
+    verify_ssl = config[DOMAIN][CONF_VERIFY_SSL]
+    token = config[DOMAIN][CONF_TOKEN]
+
+    _LOGGER.debug("host : %s", host)
+    _LOGGER.debug("verify_ssl : %s", verify_ssl)
```

Comment:
These two logging messages seem not needed.

Reply:
removed.